packages/component-library/src/components/form/mt-button/mt-button.vue

```diff
@@ -22,7 +22,13 @@
   >
     <mt-loader v-if="isLoading" size="16px" class="mt-button__loader" />
     <span class="mt-button__content" :class="contentVisibilityClass">
-      <slot />
+      <span v-if="$slots.iconFront" class="mt-button__icon mt-button__icon--front">
+        <slot name="iconFront" />
+      </span>
+      <slot></slot>
+      <span v-if="$slots.iconBack" class="mt-button__icon mt-button__icon--back">
```

Comment:
Please adjust the size of the icon depending on the size of the button.

Reply:
I've added some different icon sizes based on the button size. I've tried to keep it inline with the text size.